src/server.cc

```diff
@@ -478,6 +478,23 @@ std::atomic<uint64_t> *Server::GetClientID() {
   return &client_id_;
 }
 
+void Server::recordInstantaneousMetric() {
```

Comment:
Should be `recordInstantaneousMetrics`?  since we collect multi metrics here.

Reply:
Yes